src/EnergyPlus/DataSurfaceColors.hh

```diff
@@ -92,10 +96,10 @@ namespace DataSurfaceColors {
 
 struct SurfaceColorData : BaseGlobalStruct
 {
-    Array1D_int const defaultcolorno = Array1D_int(DataSurfaceColors::NumColors, {3, 43, 143, 143, 45, 8, 15, 195, 9, 13, 174, 143, 143, 10, 5});
-    Array1D_int DXFcolorno = Array1D_int(DataSurfaceColors::NumColors, SurfaceColorData::defaultcolorno);
+    Array1D_int const defaultcolorno = Array1D_int({0, 14}, {3, 43, 143, 143, 45, 8, 15, 195, 9, 13, 174, 143, 143, 10, 5});
+    Array1D_int DXFcolorno = Array1D_int({0, 14}, SurfaceColorData::defaultcolorno);
 
-    Array1D_string const colorkeys = Array1D_string(DataSurfaceColors::NumColors,
+    Array1D_string const colorkeys = Array1D_string({0, 14},
                                                     {"Text",
```

Comment:
What's the benefit of `{0,14}` over `ColorNo::NUM` in this constructor?

Reply:
Is this because `Array1D_string` by default starts at 1 otherwise? I think this was going to be part of the next pass, but if you want you can go ahead and change these Array1D_string const instances to constexpr std::array<std::string_view>, which will both start at zero by default and also not construct strings at runtime.

Reply:
Yes, it starts at 1 by default. I'll use `constexpr std::array<std::string_view>` so that I could go ahead next pass. Thank you for the feedback.